Pod/Classes/SideMenuTransition.swift

```diff
@@ -294,6 +294,11 @@ open class SideMenuTransition: UIPercentDrivenInteractiveTransition {
         let menuView = menuViewController?.view
         let mainView = mainViewController?.view
         
+        guard menuView != nil,
```

Comment:
1. Why do you have the check twice? 
2. You can move this line in the beginning of the function and changed it to `guard strongMenuView = menuView else { return self }`.

Reply:
You are right!